### DISCUSSION TITLE
It's just awesome

### DISCUSSION THREAD

**Discussion by chiefenne** [community/community#7410] [Copilot Conversations] [answered]
I'm using Copilot since two days. I can state that coding effort is already reduced by approximately 20-30%. Some extensions in Vscode might become obsolete, e.g. DocString Generator. 

I am just tabbing all the time. Excellent.

**Comment by SmallLemur** [comment, marked as answer]
@chiefenne thank you for the feedback, Are you still using it ? would you still say it is reducing your coding effort ?

**Reply by chiefenne** [reply]
Yes I am still using it. And I am more and more impressed/surprised what it is doing for me. It is significantly reducing my coding effort.

**Reply by SmallLemur** [reply]
that's great ! i would be very interested to hear a bit more about your experience with it, Would you be open to have a short 15-30 min discussion next week ?

**Reply by chiefenne** [reply]
yes we can do that

**Reply by SmallLemur** [reply]
Great, I will contact you on the email in your public github profile to organise it. Thank you!

**Reply by segun-luwi** [reply]
Github Copilot is just awesome, been using it for some days now and am impressed with how much it has helped reduced my development time, I can focus on other aspects of the code and Copilot will help complete my code, I can just write down my thoughts of how I expect the logic to be and Copilot will generate the code for me, really cool.

**Reply by andraz** [reply]
We are basically skipping the "google for it and copy from stack overflow" step, great way to optimize our workflow by letting AI do the redundant robotic work. 👍

**Reply by douglasrahn** [reply]
its a great start but needs to be expanded to include conversational interface in order to be a true assistant to anyone who is doing anything except writing a script from scratch. Editing, optimizing, etc is not capable today. Glad to share more anytime.

**Comment by GSPP** [comment]
I love this thing. I would not code without it.

I have all .NET solutions open in VS and VSCode at the same time. 90% of my time is spent in VSCode with Copilot. I use VS to run the code and to perform Resharper refactorings. VS is simply superior to VSCode in terms of raw C# productivity, but Copilot changes that equation.

I, too, am willing to talk or chat with you to provide direct feedback. This tool has become important to me and I'd love to have a positive impact on its progression. @SmallLemur
